.github/workflows/makefile.yml

```diff
@@ -40,10 +40,10 @@ jobs:
         path: .lake
         key: ${{ runner.os }}-lake
 
-    - name: Build AWS-LC and running ELFSage tests (Ubuntu)
+    - name: Build AWS-LC and run ELFSage tests (Ubuntu)
       if : ${{ runner.os != 'macOS' }}
       run: |
-        ./scripts/top.sh
+        ./scripts/ci_ubuntu_build_awslc.sh
```

Comment:
Can this script be executed as a part of the Make target? That'd also fix the need for the prerequisite, as you noted in the README.

Reply:
Nice. It also helped simplifying the workflow yaml file.